### DISCUSSION TITLE
Is there a way to dynamically change the properties of a spoke model in a scene whilst in hubs?

### DISCUSSION THREAD

**Discussion by lukewhitingdev** [mozilla/hubs#2850] [General Questions & Discussion] [answered]
For example importing a house into spoke and exporting it as a scene. Then when in hubs you can remove the doors etc. Is this possible with hubs and spoke?

**Comment by ocularvr** [comment, marked as answer]
You can upload your own glb model into Spoke, but cannot modify the glb once imported. To import:

Step 1 - Login to your hubs
Step 2 - Go to Spoke
Step 3 - In the tab on the bottom left, look for "My assets"
Step 4 - Click the blue button (i think same color for all) that reads "Upload..."
Step 5 - Upload your desired glb file
Step 6 - Select that mesh and place it in a scene

**Reply by lukewhitingdev** [reply]
fair enough, cheers for the reply : )
